PurchasesCoreSwift/Logging/Strings/Strings.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
     static let identity = IdentityStrings()
     static let network = NetworkStrings()
     static let offering = OfferingStrings()
-    @objc public static let purchase = PurchaseStrings()
+    static let purchase = PurchaseStrings()
```

Comment:
I think this officially means that all other properties in this file can also have the `@objc` removed

Reply:
It does 😄  But just not in this PR, ha.